docs/source/kedro_project_setup/settings.md

```diff
@@ -19,3 +21,21 @@ By default, all code in `settings.py` is commented out. When settings are not su
 | `CONFIG_LOADER_CLASS`       | `kedro.config.ConfigLoader`                       | Customise how project configuration is handled.                                                                    |
 | `CONFIG_LOADER_ARGS`        | `dict()`                                          | Keyword arguments for the `CONFIG_LOADER_CLASS` constructor.                                                       |
 | `DATA_CATALOG_CLASS`        | `kedro.io.DataCatalog`                            | Customise how the [Data Catalog](../data/data_catalog.md) is handled.                                              |
+
+## Settings in `pyproject.toml`
+The `pyproject.toml` file is defined in [PEP 518](https://peps.python.org/pep-0518/) as a configuration file to store build system requirements for Python projects.
+The `pyproject.toml` file in your project contains the metadata for your project and other tool specific configurations.
+Every Kedro project comes with a default pre-populated `pyproject.toml` file in your project root directory with the following keys specified under the `[tool.kedro]` section :
+```toml
+[tool.kedro]
+package_name = <package_name>
+project_name = <project_name>
+kedro_init_version = <kedro_version>
+```
+
+The `package_name` which should contain a [valid Python package name](https://peps.python.org/pep-0423/) for your project and `project_name` which should be a human-readable name for your project are mandatory keys.
```

Comment:
I'm looking forward for us to merge `setup.py` and `pyproject.toml` so we can unify parts of this...